client/src/partials/purchases/list/list.js

```diff
@@ -0,0 +1,180 @@
+angular.module('bhima.controllers')
+.controller('PurchaseListController', PurchaseListController);
+
+// dependencies injection
+PurchaseListController.$inject = [
+  '$translate', 'PurchaseOrderService', 'NotifyService', 'uiGridConstants', 'uiGridGroupingConstants',
+  'ModalService', '$state', 'ReceiptModal', 'SessionService', 'LanguageService',
+  'SearchFilterFormatService'
+];
+
+/**
+ * Purchase Order List Controllers
+ * This controller is responsible of the purchase list module
+ */
+function PurchaseListController ($translate, PurchaseOrder, Notify, uiGridConstants, uiGridGroupingConstants, Modal, $state, Receipts, Session, Languages, SearchFilterFormat) {
+  var vm = this;
+
+  /** gobal variables */
+  vm.filters         = { lang: Languages.key };
+  vm.formatedFilters = [];
+  vm.enterprise      = Session.enterprise;
+  vm.filterEnabled   = false;
+  vm.loading         = false;
+  vm.gridApi         = {};
+  vm.gridOptions     = {};
+
+  /** paths in the headercrumb */
+  vm.bcPaths = [
+    { label : 'TREE.PURCHASE' },
+    { label : 'TREE.PURCHASE_REGISTRY' }
+  ];
+
+  /** buttons in the headercrumb */
+  vm.bcButtons = [
+    { icon: 'fa fa-search', label: $translate.instant('FORM.BUTTONS.SEARCH'),
+      action: search, color: 'btn-default'
+    }
+  ];
+
+  var columnDefs  = [
+    { 
+        field : 'reference', 
+        displayName : 'FORM.LABELS.REFERENCE', 
+        headerCellFilter : 'translate', 
+        aggregationType: uiGridConstants.aggregationTypes.count },
```

Comment:
Since we don't have proper i18n for the grid, I encourage you to use `aggregationHideLabel: true` on all the columns that have aggregations.